README.md

```diff
@@ -183,7 +183,7 @@ app.register(mercuriusGateway, {
 - all the mercurius plugin [options](https://mercurius.dev/#/docs/api/options?id=plugin-options)
 - `gateway`: Object. Run the GraphQL server in gateway mode.
 
-  - `gateway.services`: Service[] An array of GraphQL services that are part of the gateway
+  - `gateway.services`: `Service[]` | `Function` An array of GraphQL services that are part of the gateway or a `Function` that returns a `Promise` that resolves to an array of services. **Required**.
```

Comment:
Could we update the types in `index.d.ts` (with type tests) please?

Reply:
Sure!